lib/kitchen/driver/openstack.rb

```diff
@@ -372,29 +368,6 @@ def hints_path
         Ohai::Config[:hints_path][0]
       end
 
-      def setup_ssh(server, state)
-        if config[:key_name]
-          info "Using OpenStack keypair <#{config[:key_name]}>"
-        end
-        info "Using public SSH key <#{config[:public_key_path]}>"
-        info "Using private SSH key <#{config[:private_key_path]}>"
-        state[:ssh_key] = config[:private_key_path]
-        do_ssh_setup(state, config, server) unless config[:key_name]
-      end
-
-      def do_ssh_setup(state, config, server)
```

Comment:
Ah yeah as long as the `:key_name` is declared this shouldn't be an issue either. https://github.com/test-kitchen/kitchen-openstack#public_key_path